### PR TITLE
Wm37 upgrade django

### PR DESCRIPTION
**Pull request by walshmm**

### CODE REVIEW

**Review by KedoKudo — APPROVED** [review]
Just did another round of local test with the branch and the Django upgrade works as expected.

File: environment_dasmon.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ channels:
 dependencies:
 - make
 - python=3.7
-- django=1.11.25
+- django=3.2
 - stomp.py=7.0.0
-- psycopg2=2.8.6
 - setuptools
 - postgresql
+- psycopg2-binary=2.8.6
```
Comment (rosswhitfield) [inline environment_dasmon.yml:11]: Since we now at Django 3.2 can we update `psycopg2` to the newest version?

**Review by rosswhitfield — CHANGES_REQUESTED** [review]

File: environment_dasmon.yml (hunk 1 above)
Comment (walshmm) [reply environment_dasmon.yml:11]: The latest version of psycopg2 causes some issue with the db, I forget what the error exactly was but I had to roll back for things to work.

Comment (rosswhitfield) [reply environment_dasmon.yml:11]: The issue was using old versions of Django didn't work with the newest version of psycopg2. But Dajngo 3.2 should work with the newest version of psycopg2 https://github.com/psycopg/psycopg2/issues/1293#issuecomment-863430637

Comment (rosswhitfield) [reply environment_dasmon.yml:11]: I can fix it in a separate PR.

**Pull request merged**